static/app/components/u2f/u2finterface.tsx

```diff
@@ -116,11 +121,18 @@ class U2fInterface extends Component<Props, State> {
           () => {
             const u2fResponse = this.getU2FResponse(data);
             const challenge = JSON.stringify(this.props.challengeData);
+            const stateData = this.props.stateData
+              ? JSON.stringify(this.props.stateData)
+              : this.props.stateData;
 
             if (this.state.responseElement) {
               // eslint-disable-next-line react/no-direct-mutation-state
               this.state.responseElement.value = u2fResponse;
             }
+            if (this.state.stateElement && stateData) {
+              // eslint-disable-next-line react/no-direct-mutation-state
+              this.state.stateElement.value = stateData;
+            }
```

Comment:
If we're using a view, we need to set `stateData` so we can access it in `response.POST`